core/src/test/scala/tech/beshu/ror/unit/boot/ReadonlyRestStartingTests.scala

```diff
@@ -565,7 +567,7 @@ class ReadonlyRestStartingTests extends WordSpec with Inside with MockFactory wi
                                     resourceFileName: String) = {
     (mockedCoreFactory.createCoreFrom _)
       .expects(where {
-        (config: RawRorConfig, _) => config == rorConfigFromResource(resourceFileName)
+        (config: RawRorConfig,_, MockLdapConnectionPoolProvider :LdapConnectionPoolProvider) => config == rorConfigFromResource(resourceFileName)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field name 'MockLdapConnectionPoolProvider' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/manual/scarduzio/elasticsearch-readonlyrest-plugin/pullRequest?prid=5113380)